client/blocks/comments/post-comment-list.jsx

```diff
@@ -23,28 +19,45 @@ import CommentCount from './comment-count';
 import SegmentedControl from 'components/segmented-control';
 import SegmentedControlItem from 'components/segmented-control/item';
 
+// eslint-disable
```

Comment:
`valid-jsdoc` in about a hundred ways heh.
Found a better fix.  just by creating a newline separation it won't treat it as describing the class